packages/react-query-devtools/src/__tests__/production.test.tsx

```diff
@@ -0,0 +1,8 @@
+import { describe, expect, it } from 'vitest'
+import { ReactQueryDevtools } from '..'
+
+describe('ReactQueryDevtools in production mode', () => {
+  it('should return null', () => {
+    expect(ReactQueryDevtools({})).toBeNull()
```

Comment:
how does this work? I mean, how does this test run in production mode ?

Reply:
@TkDodo, Ah, my meaning is that 'not development' almost equals `process.env.NODE_ENV='production'`. but It's not perfectly equal. so I update test case not for production mode, just for not in process.env.NODE_ENV='development' exactly.